### DISCUSSION TITLE
error shinydashboard

### DISCUSSION THREAD

**Discussion by joejnt** [teamalgoritma/community#890] [Q&A] [answered]
output$plotTrend <- renderPlotly({
    plotTrend <- df_lmbal %>%
      mutate(TglValuta = as.Date(Tanggal.Mulai, format = "%m/%d/%Y"),
             Status2 = case_when(
               Status == 1 ~ "Aktif",
               Status == 2 ~ "Lunas",
               Status == 7 ~ "Tutup Buku",
               Status == 8 ~ "Hapus Tagih"
               #TRUE ~ as.character(Status)
             ),
             YearMonth = format(as.Date(Tanggal.Mulai, format = "%m/%d/%Y"), "%Y-%m")) %>%
      filter(YearMonth >= format(Sys.Date() %m-% months(12),"%Y-%m")) %>% 
      group_by(YearMonth) %>%
      summarise(
        frek = n(),
        vol = sum(BakiDebetOutstanding)
      ) %>%
      ungroup() %>% 
      arrange(YearMonth) %>%
      mutate(
        label = glue(
          #"Keterangan : {(Status2)}"
          "Volume : {(vol)}"
        )
      )
    
    gg <- ggplot(plotTrend, aes(x = as.Date(paste0(YearMonth, "-01")), y = frek, group = 1, text = label)) +
      geom_line(colour = "green") +
      geom_point() +
      labs(title = "Tren Frekuensi per Bulan untuk Berbagai Status",
           x = "Year-Month",
           y = NULL) +
      scale_x_date(date_labels = "%Y-%m", date_breaks = "1 month") +
      theme_minimal() +
      theme(axis.text.x = element_text(angle = 45, hjust = 1))
    ggplotly(gg, tooltip = "text")
  }

Hasilnya error:
![WhatsApp Image 2024-07-25 at 16 17 49](https://github.com/user-attachments/assets/b092525c-7b7c-4ede-a19a-86964e389556)

Best regards,
JT

**Comment by saskia-dwi-ulfah** [comment, marked as answer]
Dear @joejnt 

Terima kasih atas pertanyaan yang sudah diajukan. Berdasarkan [resource](https://github.com/plotly/plotly.R/issues/2259) ini, Anda dapat mencoba untuk meng-update versi package `plotly` menjadi `4.10.4` atau versi yang paling terbaru. 

Semoga jawaban ini dapat membantu.

Best regards,
Team Algoritma

**Reply by saskia-dwi-ulfah** [reply]
Halo @joejnt  , untuk menjaga agar forum diskusi berjalan kondusif, karena tidak ada respon lanjutan setelah 7 hari dari respon terakhir, kami izin menandai pertanyaan sudah terjawab dan menutup diskusi ini. Semoga dapat dipahami dan terima kasih.
